### DISCUSSION TITLE
【求助】有人知道这个手机端为什么会出现这个问题吗

### DISCUSSION THREAD

**Discussion by Hongxuli2006** [share121/inter-knot#1727] [其他]
![绳网手机端问题](https://github.com/user-attachments/assets/bde1f708-684c-4ef1-a0fd-bc5355a40f68)
我GitHub登陆了，插件也下载了，为什么还是这样子呢
求助各位绳匠帮我解决下问题

**Comment by Hongxuli2006** [comment]
管理员或者作者看到麻烦关注下这个帖子。

**Comment by AetherLude** [comment]
换个浏览器吧，via有时候用着会出问题

**Comment by szlx2333** [comment]
我的也是这样，但是换了其他浏览器又不知道怎么装油猴T-T

**Comment by Dave-12138** [comment]
虽然我也有via，但我手机通常用Edge的——毕竟能跟电脑同步收藏夹和浏览历史。
手机上装个Edge Canary，打开右下角菜单，进设置，点<kbd>关于 Microsoft Edge</kbd>，连戳几下`Edge Canary xxx.x.xxxx.x`字样，直到解锁开发人员选项。
进入开发人员选项点`Extension install by id`，输入油猴（别的脚本管理器亦可）的id：
```
iikmkjmpaadaobahmlepeloendndfphd
```
并确认，就装上脚本管理器了。
